lfr/heterogeneous_single_assembly_3D/neutronics_standalone/neutronics.i

```diff
@@ -0,0 +1,1771 @@
+# ==============================================================================
+# Model description
+# Application : Griffin
+# ------------------------------------------------------------------------------
+# Lemont, ANL, November 14th, 2022
+# Author(s): Hansol Park, Emily Shemon
+# ==============================================================================
+# - LFR 3D Single Assembly Hot Full Power Configuration GRIFFIN neutronics input
+# - Single App
+# ==============================================================================
+# Geometry Info.
+# ==============================================================================
+half_pinpitch = 0.0067123105    # m
+fuel_r_o = 0.004318648          # m
+fuel_r_i = 0.00202042           # m
+clad_r_i = 0.004495             # m
+clad_r_o = 0.0054037675         # m
+duct_thickness = 0.003533       # m
+asmgap_thickness = 0.0018375    # m
+flat_to_flat = 0.153424         # m
+half_asmpitch = ${fparse flat_to_flat / 2 + duct_thickness + asmgap_thickness}
+
+hA = 0.1007 # Lower Core Plate
```

Comment:
I assume this block is axial heights. It will be good to add a comment like `# axial heights`. Similarly for the size and number of subdivisions next.